test/tests/functional/pbs_Rrecord_resources_used.py

```diff
@@ -378,26 +351,21 @@ def test_t5(self):
 
         # Verify that the accounting logs have Resource_List.<resource> but no
         # R records.
-        m = self.server.accounting_match(
+        self.server.accounting_match(
             msg='.*Resource_List.*', id=jid1, regexp=True)
-        self.assertNotEqual(m, None)
-        m = self.server.accounting_match(
-            msg='.*R;' + jid1 + '.*resources_used.*', id=jid1, regexp=True)
-        self.assertIsNone(m)
-        m = self.server.accounting_match(
+        msg = '.*R;' + jid1 + '.*resources_used.*'
+        self.server.accounting_match(msg, id=jid1, regexp=True,
+                                     existence=False)
```

Comment:
Oh sorry, I was waiting for a new commit.  I thought you were saying that you were changing something and were about to commit something.  If that isn't the case then I'd still like to see you be consistent in the use of either passing the first argument as the msg variable or msg=X argument.